src/__tests__/__snapshots__/BranchName.test.tsx.snap

```diff
@@ -6,9 +6,11 @@ exports[`BranchName renders consistently 1`] = `
   padding: 2px 6px;
   font-size: 12px;
   font-family: SFMono-Regular,Consolas,"Liberation Mono",Menlo,Courier,monospace;
-  color: #57606a;
+  color: #0969da;
```

Comment:
That seems like a pretty drastic change — is that expected?

Reply:
Our current BranchName component is pretty different from what's on github.com:

| github.com | Primer React |
| --- | --- |
|  ![CleanShot 2021-12-07 at 12 11 40](https://user-images.githubusercontent.com/4608155/145099257-e1a7e971-1402-4a35-a106-50d61ce54e34.png) | ![CleanShot 2021-12-07 at 12 10 41](https://user-images.githubusercontent.com/4608155/145099204-694ba510-4298-493c-9734-fa114ee10b6b.png) |

This change fixes that.